front/src/common/Map/Search/MapSearchStation.jsx

```diff
@@ -126,11 +127,10 @@ export default function MapSearchStation(props) {
       <div className="search-results">
         {searchResults &&
           orderResults(searchResults).map((result) => (
-            <div className="mb-1">
+            <div className="mb-1" key={`${result.trigram}${result.yardname}${result.uic}`}>
```

Comment:
Add a specific item to avoid non unique key, thanx !
(eg : `mapSearchStation-${result...`)

Reply:
Done !!